lib/opencensus/trace/span_builder.rb

```diff
@@ -435,7 +435,7 @@ def initialize span_context, skip_frames: 0
         @links = []
         @status_code = nil
         @status_message = nil
-        @stack_trace = caller_locations(skip_frames + 2)
+        @stack_trace = caller_locations(skip_frames + 2).map(&:to_s)
```

Comment:
One thing we need to consider about this fix is whether or cost of the loop here offsets the advantages of reducing `caller_locations` to strings at this level.